### ISSUE TITLE
Pre-formatted text is not visible if text wraps

### ISSUE THREAD

**Issue by merelister** [pitr/gemini-ios#35]
The text in lines that start with ``` do not wrap, so part of the text is cut off.
![image0](https://user-images.githubusercontent.com/24901233/135388588-c4ff34da-5b63-4a08-b5b6-e741962f3696.png)

You can drag to scroll horizontally on the line of text, but the portion that was cut off will not be visible. 
![image1](https://user-images.githubusercontent.com/24901233/135388625-e215be4f-d7c5-4991-a72c-9fecc2062418.png)

However, if you highlight the text and copy/paste it elsewhere, the hidden text can be copied too.

(screenshots from gemini.circumlunar.space/docs/cheatsheet.gmi)

**Comment by pitr** [comment]
yes unfortunately this is a known issue. I have spent a lot of time trying to debug it and understand the root cause. I know that if I disable [the custom font](https://github.com/pitr/gemini-ios/tree/master/Client/Assets/Fonts) then the problem disappears, but then the result is not as pretty, and headers on pages like gemini://gemini.cyberbot.space/ don't render well. I am tempted to just bite the bullet and disable the custom font.
